service/dynamodb/examples_test.go

```diff
@@ -8,1495 +8,536 @@ import (
-		ProjectionExpression:   aws.String("ProjectionExpression"),
-		ReturnConsumedCapacity: aws.String("ReturnConsumedCapacity"),
-		ScanFilter: map[string]*dynamodb.Condition{
-			"Key": { // Required
-				ComparisonOperator: aws.String("ComparisonOperator"), // Required
-				AttributeValueList: []*dynamodb.AttributeValue{
-					{ // Required
-						B:    []byte("PAYLOAD"),
-						BOOL: aws.Bool(true),
-						BS: [][]byte{
-							[]byte("PAYLOAD"), // Required
-							// More values...
-						},
-						L: []*dynamodb.AttributeValue{
-							{ // Required
-							// Recursive values...
-							},
-							// More values...
-						},
-						M: map[string]*dynamodb.AttributeValue{
-							"Key": { // Required
-							// Recursive values...
-							},
-							// More values...
-						},
-						N: aws.String("NumberAttributeValue"),
-						NS: []*string{
-							aws.String("NumberAttributeValue"), // Required
-							// More values...
-						},
-						NULL: aws.Bool(true),
-						S:    aws.String("StringAttributeValue"),
-						SS: []*string{
-							aws.String("StringAttributeValue"), // Required
-							// More values...
-						},
-					},
-					// More values...
-				},
-			},
-			// More values...
-		},
-		Segment:       aws.Int64(1),
-		Select:        aws.String("Select"),
-		TotalSegments: aws.Int64(1),
-	}
-	resp, err := svc.Scan(params)
+// To list tables
+//
+// This example lists all of the tables associated with the current AWS account and
+// endpoint.
+func ExampleDynamoDB_ListTables_shared00() {
+	svc := dynamodb.New(session.New())
+	input := &dynamodb.ListTablesInput{}
 
+	result, err := svc.ListTables(input)
 	if err != nil {
-		// Print the error, cast err to awserr.Error to get the Code and
-		// Message from an error.
-		fmt.Println(err.Error())
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case dynamodb.ErrCodeInternalServerError:
+				fmt.Println(dynamodb.ErrCodeInternalServerError, aerr.Error())
+			default:
+				fmt.Println(aerr.Error())
+			}
+		} else {
+			// Print the error, cast err to awserr.Error to get the Code and
+			// Message from an error.
+			fmt.Println(err.Error())
+		}
 		return
 	}
 
-	// Pretty-print the response data.
-	fmt.Println(resp)
+	fmt.Println(result)
 }
 
-func ExampleDynamoDB_TagResource() {
-	sess := session.Must(session.NewSession())
-
-	svc := dynamodb.New(sess)
-
-	params := &dynamodb.TagResourceInput{
-		ResourceArn: aws.String("ResourceArnString"), // Required
-		Tags: []*dynamodb.Tag{ // Required
-			{ // Required
-				Key:   aws.String("TagKeyString"),   // Required
-				Value: aws.String("TagValueString"), // Required
-			},
-			// More values...
+// To add an item to a table
+//
+// This example adds a new item to the Music table.
+func ExampleDynamoDB_PutItem_shared00() {
+	svc := dynamodb.New(session.New())
+	input := &dynamodb.PutItemInput{
+		Item: map[string]*dynamodb.AttributeValue{
+			"AlbumTitle": {},
+			"Artist":     {},
+			"SongTitle":  {},
```

Comment:
I think something in the example generator is preventing `AttributeValue` fields from getting rendered.  Or is these just missing from the model?